monetdbe/_cffi/internal.py

```diff
@@ -180,11 +169,13 @@ def open(self) -> monetdbe_database:
                 lib.monetdbe_close(connection)
             else:
                 error = errors.get(result_code, "unknown error")
-            raise exceptions.OperationalError(f"Failed to open database: {error} (code {result_code})")
+            msg = "Failed to open database: {error} (code {result_code})".format(error=error, result_code=result_code)
```

Comment:
the `.format()` is quite old and not really the preferred string formatting method anymore for py>3.5

Reply:
👍 will be in the next push.